kolibri/plugins/coach/assets/src/views/create-exam-page/index.vue

```diff
@@ -1,12 +1,12 @@
 <template>
 
   <div>
-    <h1>{{ $tr('createNewExam') }}</h1>
+    <h1>{{ `${$tr('createNewExam')} ${currentChannel.name}` }}</h1>
```

Comment:
A bit confusing to read, and I'm not sure this will stand up to translation very well since it's forcing and ordering of the words